doc-content/enterprise-only/openshift/customimage-con.adoc

```diff
@@ -1,12 +1,11 @@
 [id='customimage-con_{context}']
-= Creating custom images for {CENTRAL} and {KIE_SERVER}
+= Creating custom images for {KIE_SERVER}
 
-You can create custom images to add files to the {CENTRAL} and {KIE_SERVER} deployments. You must then push the images to your own Docker registry. When deploying {PRODUCT}, you can configure the operator to use the custom images.
+You can create custom images to add files to {KIE_SERVER} deployments. You must then push the images to your own container registry. When deploying {PRODUCT}, you can configure the operator to use the custom images.
```

Comment:
I suggest to remove "then" from: You must then push the images to your own container registry.